### PR TITLE
[DOCS-6138] Revise Single Step Instrumentation Beta Banner

### PR DESCRIPTION
**Pull request by brett0000FF**
### What does this PR do?

Revise beta banner wording to
- Give more context.
- Make a stronger statement about this new feature.
- Highlight key value prop.

### Motivation

https://datadoghq.atlassian.net/browse/DOCS-6138

* Feedback from Irene.
* Reviewing heat maps in RUM and Metabase analytics for this content, it appears that users needed more context before diving into the Single Step APM Instrumentation page.
* https://metabase-analytics.us1.prod.dog/dashboard/36736-apm-docs-dashboard?relative_date=past30days

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.
- [ ] Check that `cache_enabled` is set to `true` in the `pull_config_preview.yaml` file

### CODE REVIEW

File: content/en/tracing/trace_collection/_index.md [hunk 1]
```diff
@@ -22,9 +22,9 @@ To configure your application to send traces to Datadog:
 
 ## Step 1 - Configure the Datadog Agent for APM
 
-<div class="alert alert-info"><strong>Beta</strong>: If you use <a href="/tracing/trace_collection/single-step-apm">Single Step APM Instrumentation</a> when you install or update your Agent to the latest version, <strong>you're done!</strong> With this option, APM is configured in the Agent and all your services on that host, VM, or Docker container are automatically instrumented and report APM traces to Datadog. <a href="/tracing/services">Start exploring!</a></div>
+<div class="alert alert-info"><strong>Beta</strong>: You can enable APM when installing the Agent with  <a href="/tracing/trace_collection/single-step-apm">Single Step APM Instrumentation</a>! Run a one-line install command to automatically enable APM and instrument all of your services on the Linux host, VM, or container. After that, you're ready to <a href="/tracing/services">start exploring</a>!</div>
```
Comment (jhgilbert) [inline content/en/tracing/trace_collection/_index.md:25]:
```suggestion
<div class="alert alert-info"><strong>Beta</strong>: You can enable APM when installing the Agent with  <a href="/tracing/trace_collection/single-step-apm">Single Step APM Instrumentation</a>. Run a one-line install command to automatically enable APM and instrument all of your services on the Linux host, VM, or container. After that, you're ready to <a href="/tracing/services">start exploring</a>!</div>
```

Comment (jhgilbert) [reply content/en/tracing/trace_collection/_index.md:25]: Two in a row felt a little intense, but you know the desired tone better than I do!

File: content/en/tracing/trace_collection/_index.md [hunk 2]
```diff
@@ -22,9 +22,9 @@ To configure your application to send traces to Datadog:
...
 
-If you don't use the Single Step APM Instrumentation option, APM-specific configurations are required on both the Tracer and Agent to ensure that traces can be properly received from containerized, serverless, or certain other environments. Ensure you have followed instructions for both components. 
+If you don't use Single Step APM Instrumentation, APM-specific configurations are required on both the Tracer and Agent to ensure that traces can be properly received from containerized, serverless, or certain other environments. Ensure you have followed instructions for both components.
```
Comment (jhgilbert) [inline content/en/tracing/trace_collection/_index.md:27]:
```suggestion
If you don't use Single Step APM Instrumentation, APM-specific configurations are required on both the Tracer and Agent to ensure that traces can be received from certain environment types, such as containerized or serverless. Ensure you have followed instructions for both components. 
```

Comment (jhgilbert) [reply content/en/tracing/trace_collection/_index.md:27]: Again just a suggestion, "certain other" felt odd to me but I know it was already there.

**Review by jhgilbert — APPROVED** [review]
Approved with optional suggestions. Thanks!

**Comment by brett0000FF** [comment]
@jhgilbert - Great suggestions. Thanks!

**Pull request merged**